letsencrypt/scripts/main.py

```diff
@@ -96,6 +96,10 @@ def create_parser():
     add("--apache-init-script", default="/etc/init.d/apache2",
         help=config_help("apache_init_script"))
 
+    add("--dns-server", default="localhost", help=config_help("dns_server"))
+    add("--dns-server-port", default=53, help=config_help("dns_server_port"))
+    add("--dns-tsig-keys", default=[], type=split_tsig_keys, help=config_help("dns_tsig_keys"))
```

Comment:
`default=[]` seems fragile here (it's mutable)... Also, `split_tsig_keys` returns a 3-tuple, so that types don't really match up. I guess it should be `default=None` (which is the default, so can be omitted), and you should also test in the authenticator that `config.dns_tsig_keys is not None`